src/transform-sdk/js/schema_registry_example/src/index.ts

```diff
@@ -0,0 +1,41 @@
+import { onRecordWritten, OnRecordWrittenEvent, RecordWriter } from "@redpanda-data/transform-sdk"
+import { newClient, decodeSchemaID } from "@redpanda-data/transform-sdk-sr";
+import { Type } from "avsc/lib/index.js"
+import { Buffer } from "buffer"
+
+var sr_client = newClient();
+const schema = {
+    type: "record",
+    name: "Example",
+    fields: [
+        { "name": "a", "type": "long", "default": 0 },
+        { "name": "b", "type": "string", "default": "" }
+    ]
+};
+
+const subj_schema = sr_client.createSchema(
+    "avro-value",
+    {
+        schema: JSON.stringify(schema),
+        format: 0,//SchemaFormat.Avro,
```

Comment:
We should probably fix this...

Reply:
ah, right. yeah, i'll follow up with something.